charts/gxf/templates/service.yaml

```
@@ -1,5 +1,8 @@
 {{- $releaseName := .Release.Name -}}
 {{- $servicePort := .Values.service.port -}}
+{{ if .Values.httpdSidecar.enabled }}
+  {{- $servicePort = 443 -}}
+{{ end }}
```

Overriding servicePort is not possible after enabling httpdSidecar.
```suggestion
{{ if .Values.httpdSidecar.enabled }}
  {{- $servicePort := .Values.service.port | default 443 -}}
{{- else }}
  {{- $servicePort := .Values.service.port | default 80 -}}
{{ end }}
```